packages/overlay/src/overlay-stack.ts

```diff
@@ -478,7 +509,7 @@ export class OverlayStack {
     private async hideAndCloseOverlay(
         overlay?: ActiveOverlay,
         animated?: boolean,
-        clickAway?: boolean
+        shouldNotReceiveFocus?: boolean
```

Comment:
nit: could this name be more obvious / how could we avoid negating a boolean? What do you think of `skipFocus`?